tjpcov/covariance_fourier_ssc.py

```diff
@@ -28,6 +27,38 @@ def __init__(self, config):
 
         self.ssc_conf = self.config.get("SSC", {})
 
+    def _get_sigma2_B(self, cosmo, a_arr, masks={}):
```

Comment:
Don't use an empty dictionary as default value of a function, use `None`. An empty dictionary can lead to unexpected results: https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python.

Reply:
See comment below; I would make this function `_get_sigma2_B(self, cosmo, a_arr, tr)`

Reply:
Got it. Edited in new push